samples/client/others/rust/reqwest-oneof-discriminator/src/models/fruit_one_of_enum_mapping_disc.rs

```diff
@@ -0,0 +1,30 @@
+/*
+ * fruity
+ *
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * 
+ * Generated by: https://openapi-generator.tech
+ */
+
+
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "fruitType")]
```

Comment:
this was `fruit_type` prior to changing `getPropertyName()` to `getPropertyBaseName()`